dropshot/src/handler.rs

```diff
@@ -1119,12 +1119,17 @@ pub trait HttpTypedResponse:
      * trait method to allow callers to avoid redundant type specification.
      */
     fn for_object(body_object: &Self::Body) -> HttpHandlerResult {
-        let serialized = serde_json::to_string(&body_object)
-            .map_err(|e| HttpError::for_internal_error(e.to_string()))?;
-        Ok(Response::builder()
-            .status(Self::STATUS_CODE)
-            .header(http::header::CONTENT_TYPE, CONTENT_TYPE_JSON)
-            .body(serialized.into())?)
+        let response = Response::builder().status(Self::STATUS_CODE);
```

Comment:
@davepacheco I'd love to get your eyes on this in particular; I don't love it, but I think maybe it's better than not doing it. In particular, I think casting is icky, but I don't know how else to see if we've got an Empty type. Alternatively, perhaps we don't make `Empty` pub.

Reply:
I kind of like not making `Empty` pub and saying that you should use a different `HttpResponseOkEmpty`...but I hope someone will push back if that seems excessively pedantic.  I'd even consider renaming `HttpResponseOk` to `HttpResponseOkBody`, though that does seem pretty annoying for existing consumers.

Reply:
Well it turns out that `Empty` must be pub because it's used as the `Body` type param for some `pub` response types. But I made it `#[doc(hidden)]` to discourage its use... which would otherwise fail at runtime. I didn't rename the type because--as you said--it would be annoying to consumers.